### PR TITLE
tentacle_manager integration 

### PR DESCRIPTION
**Pull request by GuillaumeDSM**
requires https://github.com/Drakkar-Software/OctoBot-Commons/pull/28

### CODE REVIEW

File: octobot_backtesting/abstract_backtesting_test.py [hunk 1]
```diff
@@ -215,10 +217,11 @@ def _register_only_strategy(self, strategy_evaluator_class):
...
             from octobot_evaluators.evaluator.strategy_evaluator import StrategyEvaluator
             from octobot_evaluators.constants import CONFIG_EVALUATOR
-            for evaluator_name in self.config[CONFIG_EVALUATOR]:
+            tentacles_activation = get_tentacles_activation(self.tentacles_setup_config)
+            for evaluator_name in tentacles_activation:
                 if get_class_from_string(evaluator_name, StrategyEvaluator, Strategies,
                                          evaluator_parent_inspection) is not None:
-                    self.config[CONFIG_EVALUATOR][evaluator_name] = False
-            self.config[CONFIG_EVALUATOR][strategy_evaluator_class.get_name()] = True
+                    tentacles_activation[evaluator_name] = False
```
Comment (Herklos) [inline octobot_backtesting/abstract_backtesting_test.py]: :+1:

File: octobot_backtesting/octobot_backtesting.py [hunk 2]
```diff
@@ -143,6 +148,7 @@ def _log_remaining_object_error(self, obj, expected, actual):
         for exchange_class_string in self.symbols_to_create_exchange_classes.keys():
             exchange_builder = create_exchange_builder(self.backtesting_config, exchange_class_string) \
                 .has_matrix(self.matrix_id) \
+                .use_tentacles_setup_config(self.tentacles_setup_config) \
```
Comment (Herklos) [inline octobot_backtesting/octobot_backtesting.py]: :+1:

File: octobot_backtesting/strategy_optimizer/strategy_optimizer.py [hunk 3]
```diff
@@ -38,18 +38,19 @@ class StrategyOptimizer:
...
         try:
             from octobot_evaluators.api.evaluators import create_evaluator_classes
             from octobot_evaluators.evaluator.strategy_evaluator import StrategyEvaluator
             from octobot_trading.api.modes import init_trading_mode_config, get_activated_trading_mode
             from tentacles.Evaluator import Strategies
-            init_trading_mode_config(self.config, CONFIG_TRADING_FILE_PATH)
+            init_trading_mode_config(self.config)
             create_evaluator_classes(self.config)
-            self.trading_mode = get_activated_trading_mode(self.config)
+            self.trading_mode = get_activated_trading_mode(self.config, tentacles_setup_config)
```
Comment (Herklos) [inline octobot_backtesting/strategy_optimizer/strategy_optimizer.py]: Do we still need `self.config` in `get_activated_trading_mode` with the new `tentacles_setup_config` param ?

Comment (GuillaumeDSM) [reply octobot_backtesting/strategy_optimizer/strategy_optimizer.py]: Yes, we need it to access advanced classes

Comment (Herklos) [reply octobot_backtesting/strategy_optimizer/strategy_optimizer.py]: Are they still stored in `config` ?

Comment (GuillaumeDSM) [reply octobot_backtesting/strategy_optimizer/strategy_optimizer.py]: We didn't change that so yep, they are set there by the advanced manager

Comment (Herklos) [reply octobot_backtesting/strategy_optimizer/strategy_optimizer.py]: OK do you think it's something that should be changed with the new tentacle manager ?

Comment (GuillaumeDSM) [reply octobot_backtesting/strategy_optimizer/strategy_optimizer.py]: For now the tentacle manager is only about tentacles maintenance and configuration I'm not sure if it makes sense to add advanced classes management to it. I think we should wait a bit to see if we even need these advanced classes as they are currently handled.

**Pull request merged**